package.json

```diff
@@ -14,69 +14,69 @@
     "e2e": "ng e2e"
   },
   "dependencies": {
-    "@angular/animations": "16.2.7",
-    "@angular/cdk": "16.2.7",
-    "@angular/common": "16.2.7",
-    "@angular/compiler": "16.2.7",
-    "@angular/core": "16.2.7",
-    "@angular/forms": "16.2.7",
-    "@angular/platform-browser": "16.2.7",
-    "@angular/platform-browser-dynamic": "16.2.7",
-    "@angular/router": "16.2.7",
+    "@angular/animations": "17.0.9",
```

Comment:
maybe better update to 17.1.0 just released

Reply:
The rest of libraries have been updated to 17.0.9. It's better to have the same version for all